pkg/compiler/codegen.go

```diff
@@ -852,7 +852,7 @@ func (c *codegen) Visit(node ast.Node) ast.Visitor {
 				// This way even non-pointer struct fields will be copied.
 				emit.Opcodes(c.prog.BinWriter, opcode.NEWARRAY0,
 					opcode.DUP, opcode.ROT, opcode.APPEND,
-					opcode.PUSH0, opcode.PICKITEM)
+					opcode.POPITEM)
```

Comment:
Wouldn't it reverse item order?

Reply:
What do you mean? We have array of 1 element -- the struct we need to copy.